src/System.Management.Automation/engine/parser/Parser.cs

```diff
@@ -4532,6 +4532,8 @@ private StatementAst EnumDefinitionRule(List<AttributeBaseAst> customAttributes,
             //G      enum-member  new-lines:opt
             //G      enum-member-list   enum-member
 
+            TypeCode validUnderlyingTypeCodes = TypeCode.Byte | TypeCode.Int16 | TypeCode.Int32 | TypeCode.Int64 | TypeCode.SByte | TypeCode.UInt16 | TypeCode.UInt32 | TypeCode.UInt64;
```

Comment:
It should be `readonly static`.

Reply:
`const` would be preferred if it's allowed.

Reply:
@iSazonov the type constraint for enums is defined as part of the Common Language Specification in ECMA-335 and hasn't changed since 2001 - I think it's unlikely we'll ever have to update it

Reply:
Moreover - it's a public api, the values can't change. This is why we also specify enum values in PowerShell in places where it might not seem necessary, e.g. `TokenKind`.